src/components/TransactionItem/TransactionItemCell/index.tsx

```diff
@@ -81,6 +85,64 @@ const isDaoCell = (cellType: string) => {
   return isDaoDepositCell(cellType) || isDaoWithdrawCell(cellType)
 }
 
+const NervosDAOAddress = ({ cell, address }: { cell: State.Cell; address: string }) => {
+  const { app } = useContext(AppContext)
+  const WithdrawInfo = (
+    <WithdrawInfoPanel longTitle={app.language === 'en'}>
+      <div>
+        <div className="withdraw__info_title">{`${i18n.t('nervos_dao.deposit_capacity')}: `}</div>
+        <div className="withdraw__info_content">
+          <span>{localeNumberString(shannonToCkb(cell.capacity))}</span>
+          <span> CKB</span>
```

Comment:
it is generally recommended to use variables to save unit, in case of unit change.

Reply:
OK. I will update it.